extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -316,15 +423,10 @@ export async function getCompletionItemsFromSpecs(
 		}
 		filesRequested = true;
 		foldersRequested = true;
```

Comment:
I don't have the bandwidth today to think about this, but wonder if always setting these like this is now incorrect?

Reply:
https://github.com/microsoft/vscode/issues/240033